Default (OSX).sublime-settings

```diff
@@ -0,0 +1,4 @@
+{
+  "use_local_url": false,
+  "local_url": "file:///Applications/Unity/Documentation/en/ScriptReference/30_search.html"
```

Comment:
@talitore Sorry for the delay. I’m using Unity 5, I guess they moved the documentation. If the url doesn’t match you can always override it inside your `Unity3DReference.sublime-config`.